web/tests/test_chapters.py

```
@@ -0,0 +1,247 @@
+        # Add a resource
+        resource_data = {
+            "title": "New Test Resource",
+            "description": "A new test resource description",
+            "resource_type": "template",
+            "external_url": "https://example.com/new-resource",
+        }
+
+        response = self.client.post(reverse("add_chapter_resource", args=[self.chapter.slug]), resource_data)
+
+        # Check that the resource was created
+        self.assertTrue(ChapterResource.objects.filter(title="New Test Resource").exists())
```

Ruff (0.8.2)

Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)